zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/archiver/ElasticsearchRepository.java

```diff
@@ -184,7 +184,6 @@ public CompletableFuture<Void> reindexDocuments(
 
   @Override
   public void close() throws Exception {
-    // TODO: verify if I close the transport that any pending futures are cancelled
```

Comment:
❓ You mentioned that you did a small test for this correct?

Reply:
Yeah, just a quick and dirty unit test. I don't think it's worth keeping, but it was just to confirm for myself that this behaves as expected.